libs/ojsonnet/Manifest_jsonnet.mli

```diff
@@ -1,7 +0,0 @@
-exception Error of string * Tok.t
-
-(* TODO: at some point we probably would prefer an AST_generic.program
- * which could track the origin of tokens through import, eval, and
- * manifestation and that we could pass to osemgrep to evaluate the rules.
- *)
-val manifest_value : Value_jsonnet.value_ -> JSON.t
```

Comment:
We could keep Manifest_jsonnet.ml, but just call the one in Eval_jsonnet.ml, so at least we mimic more the jsonnet spec in the way the files are organized.